src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,25 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RuntimeException("User can not be null");
```

Comment:
Consider using a more specific exception rather than a generic RuntimeException. For example, IllegalArgumentException or a custom exception like RegistrationException would be more informative.